packages/admin/admin-theme/src/typographyOptions.ts

```diff
@@ -13,39 +14,75 @@ export const typographyOptions: TypographyOptions = {
     ...fontWeights,
     h1: {
         fontFamily,
-        fontSize: 55,
-        lineHeight: "64px",
+        fontSize: 36,
+        lineHeight: "42px",
         fontWeight: fontWeights.fontWeightLight,
+
+        "@media (min-width: 900px)": {
```

Comment:
Could we use the theme's breakpoint values here?

Reply:
Currently not, see: https://github.com/vivid-planet/comet/pull/1808#discussion_r1520921266 
Will be made possible with COM-568